users/ericgebhart/ericgebhart.h

```diff
@@ -250,8 +250,8 @@ void x_reset (qk_tap_dance_state_t *state, void *user_data);
 // Since our quirky block definitions are basically a list of comma separated
 // arguments, we need a wrapper in order for these definitions to be
 // expanded before being used as arguments to the LAYOUT_xxx macro.
-#if (!defined(LAYOUT) && defined(KEYMAP))
-#define LAYOUT KEYMAP
```

Comment:
This shouldn't be changed.

Reply:
It doesn't compile, at least not for me, without changing it.

With these changes, there is no `KEYMAP` macro. There's only `LAYOUT_ergodox`, `LAYOUT_ergodox_80`, and the `pretty` versions of those to choose from. (I need to make another commit to delete the alias lines in `ergodox_ez.h`.)

If you have a better way of achieving the desired result, I'm all ears.

Reply:
Yeah, because the keymap.c should have `LAYOUT_ergodox_wrapper` rather than `LAYOUT_wrapper`